tests/baselines/reference/declFileTypeAnnotationVisibilityErrorAccessors.js

```diff
@@ -287,4 +287,5 @@ declare module m {
         foo114: m2.public2;
         foo115: m2.public2;
     }
+    export {};

```

Comment:
Why do you need this here? exported class above is enough to make things public and private ?

Reply:
Not in a declaration file (or other ambient context) it's not. :smile: Just like at the top level, unless there's an export declaration or export assignment, all the things are implicitly exported, export modifier or no.

Reply:
oh I see :)